src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -93,6 +93,9 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
     @TestVisible
     private Integer rdDiffCount = 0;
 
+    @TestVisible
+    private Integer fixedLengthRdCount = 0;
```

Comment:
Since other RD vars start with `rd`, let's rename this var to `rdFixedLengthCount`?